### PR TITLE
fix: added additional response codes

### PR DESCRIPTION
**Pull request by mattp-swirldslabs**
**Description**:
Added additional response codes to cover cases encountered on the Block Node side.

**Checklist**

- [x] Is the HIP updated or a new HIP created for this change.
- [ ] Is An identical PR created to match changes in the copy in `hedera-services`.

### CODE REVIEW

**Review by jsync-swirlds — APPROVED** [review]
HIP is updated to match.
Still need a mirror PR to hedera-services

File: block/block_service.proto [hunk 1]
```diff
@@ -595,6 +607,12 @@ enum SubscribeStreamResponseCode {
      * `end_block_number` field to a valid end block.
      */
     READ_STREAM_INVALID_END_BLOCK_NUMBER = 4;
+
+    /**
+     * The requested stream is not available.<br/>
+     * The client MAY retry again later.
+     */
+    READ_STREAM_NOT_AVAILABLE = 5;
```
Comment (mhess-swl) [inline block/block_service.proto:615]: `READ_` is implied by 'subscription' imo, but not a big deal

**Review by mhess-swl — APPROVED** [review]

File: block/block_service.proto (hunk 1 above)
Comment (mattp-swirldslabs) [reply block/block_service.proto:615]: Makes sense. I added the READ_ prefix to align with others in `SubscribeStreamResponseCode`

**Comment by mattp-swirldslabs** [comment]
Closing this PR because of git DCO issues. I'll open a new PR

**Pull request closed without merging**